sdk/identity/azure-identity-broker/src/main/java/com/azure/identity/broker/InteractiveBrowserBrokerCredentialBuilder.java

```diff
@@ -46,7 +51,14 @@ public class InteractiveBrowserBrokerCredentialBuilder extends InteractiveBrowse
      * @return An updated instance of this builder with the interactive browser broker configured.
      */
     public InteractiveBrowserBrokerCredentialBuilder setWindowHandle(long windowHandle) {
-        this.identityClientOptions.setBrokerWindowHandle(windowHandle);
+        try {
+            Field field = CredentialBuilderBase.class.getDeclaredField("identityClientOptions");
+            field.setAccessible(true);
```

Comment:
I'd check with @JonathanGiles if reflection is the best approach here.
My concern is around portability to different Java environments and platforms.

In this case, would it be better to have protected methods on the parent builder, so we can invoke them via super here to do this configuration ?

Reply:
changed approach. For posterity, getting rid of a protected field was the goal here.